### PR TITLE
feat(t27): CRUD for map pools

### PR DESCRIPTION
**Pull request by AlekseySamokhin**
Added the ability to add and update card pools.
For add map pool, I created a button 'add map pool' on the page on click, which opens a modal window with an extension to add a name.
For update map pool, I created a new page, the transition to which is carried out by clicking on the map pool in table.
Please let me know if something is wrong.

### CODE REVIEW

File: modules/mapPools/api/mutations/createMapPool.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { gql } from '@apollo/client';
```
Comment (killmenot) [inline modules/mapPools/api/mutations/createMapPool.ts:1]: the module should have the same name like we have on backend

modules/maps/api
 - mapPools (folder)
    - mutations.ts
    - queries.ts
 - maps (folder)
  ....

File: modules/mapPools/components/forms/FormCreateMapPool.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    reset,
+  } = useForm<ITypeDataForm>({
+    defaultValues: {
+      mapPoolName: '',
+    },
+    resolver: yupResolver(createMapPoolSchema),
+  });
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
```
Comment (killmenot) [inline modules/mapPools/components/forms/FormCreateMapPool.tsx:48]: useTimeout always bad

**Review by killmenot — CHANGES_REQUESTED** [review]
please see inline comments

**Comment by AlekseySamokhin** [comment]
I changed the structure according to your comments. And also removed from the code setTimeout();

**Comment by AlekseySamokhin** [comment]
@killmenot CRUD for maps to do or wait for a conversation on Wednesday?

**Comment by killmenot** [comment]
You can work on it

**Pull request merged**